README.md

```diff
@@ -24,7 +24,7 @@ and the API is subject to change._
    ```
    This will put a copy of the `ion` executable in `~/.cargo/bin`.
 
-   **If this step fails:** You're likely missing one of `ion-c`'s dependencies. Make sure you have `cmake`, `gcc`, `g++`, and `libc++` installed.
+   **If this step fails:** You're likely missing one of `ion-c`'s dependencies. Make sure you have `cmake`, `gcc`, `g++` and `clang` installed. On Debian-based Linux distributions, the only required dependencies are `cmake` and `clang`.
```

Comment:
Something about this isn't quite right; we shouldn't require both `gcc` _and_ `clang`. I don't want this to block the PR, though, so I'll open an issue to track it.

Reply:
Opened #14.

Reply:
Agreed, that that doesn't seem right. I did only require `clang` for the Debian build but hadn't done much work to know if `gcc` was needed when building on macOS/Windows.